src/components/Battles.ts

```diff
@@ -189,6 +189,7 @@ export interface IPokemonBattleOptions {
      * Things displayed in the battle.
      */
     things: IBattleThings;
+
```

Comment:
Revert unrelated change?

Reply:
Will do, I think I left that in when I deleted a property there